src/main/java/frc/robot/constants/FLYWHEEL.java

```diff
@@ -3,12 +3,25 @@
 import edu.wpi.first.math.util.Units;
 
 public final class FLYWHEEL {
-  public static double gearRatio = 1.0;
+  public static double gearRatioBottom = 1.0;
+  public static double gearRatioTop = 1.0;
+
+  public static final int kSlotIdx = 0;
+
+  public static final double kP = 10;
+  public static final double kI = 0.0;
+  public static final double kD = 1;
+
+  public static final double kS = 0.0;
+  public static final double kV = 0.0;
+  public static final double kA = 0.0;
+
+  public static final double shooterangle = 35;
```

Comment:
Usually this should be in radian. Use Unit.DegreestoRadians(35) here for functionality